contracts/contracts/marketplace/v00/Marketplace.sol

```diff
@@ -182,6 +185,11 @@ contract V00_Marketplace is Ownable {
         public
         payable
     {
+        require(
+            _affiliate == 0x0 || allowedAffiliates[address(this)] || allowedAffiliates[_affiliate],
```

Comment:
should we allow `0x0`?

Reply:
`address(this)` isn't obvious for "whitelist all". maybe define a constant above (`ALL_AFFILIATES`?) and use it here?

Reply:
I've pushed a change that adds the constant. It shouldn't use any gas, because constants don't get storage slots.

Reply:
Regarding the 0x0, we could remove the check and instead add 0x0 to the whitelist in the constructor. That would give us control over whether no affiliate should be allowed